doc/guides/writing-tests.md

```diff
@@ -264,7 +264,8 @@ These imported tests will be wrapped like this:
 
 ```js
 /* eslint-disable */
-/* WPT Refs:
+/* The following tests are copied from WPT, modifications to them should be upstreamed first.
```

Comment:
I can certainly make the change, the recommendation for that particular comment is as per discussion [here](https://github.com/nodejs/node/pull/12798#pullrequestreview-36256522) cc @joyeecheung for feedback as well.